### DISCUSSION TITLE
Disable WebRTC playback in Ant Media Server

### DISCUSSION THREAD

**Discussion by yashtandon113** [ant-media/Ant-Media-Server#5219] [FAQs] [answered]
In my use case, I only want to use HLS. Is there any way to disable the WebRTC playback to save resources as there is no option to disable it on the dashboard?

**Comment by Mohit-3196** [comment, marked as answer]
By default, AMS allows WebRTC playback.
To disable WebRTC playback for your Ant Media Server application, follow these steps:
1. Navigate to the specific application settings directory. For example:
`cd /usr/local/antmedia/webapps/LiveApp/WEB-INF`
2. Open the `red5-web.properties` file for editing:
`sudo vim red5-web.properties'
3. Locate the `settings.webRTCEnabled` property and set it to `false`:
`settings.webRTCEnabled=false`
4. Save the changes and exit the editor.
5. Restart your Ant Media Server for the changes to take effect:
`sudo service antmedia restart`

After following these steps, the WebRTC playback feature will be disabled in your Ant Media Server application.

If you are using AMS in cluster mode, you can follow [these steps](https://github.com/orgs/ant-media/discussions/5161) to change the settings.
